DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -77,7 +77,8 @@ echo RUN = $RUNNUMBER
 if [[ $RUNNUMBER -ge 521889 ]]; then
   export ARGS_EXTRA_PROCESS_o2_ctf_reader_workflow="$ARGS_EXTRA_PROCESS_o2_ctf_reader_workflow --its-digits --mft-digits"
   export DISABLE_DIGIT_CLUSTER_INPUT="--digits-from-upstream"
-  MAXBCDIFFTOMASKBIAS_ITS="ITSClustererParam.maxBCDiffToMaskBias=10"
+  MAXBCDIFFTOMASKBIAS_ITS="ITSClustererParam.maxBCDiffToMaskBias=-10"    # this explicitly disables ITS masking
```

Comment:
No, it should be safe. In principle, we could do this for any runNumber (and even change this as default in the code).

Pinging to @mcoquet642 in case they want it also for the MFT.

Reply:
Provided that the same applies to the MFT clusterizer I updated the PR.

Reply:
Thanks a lot

Reply:
Perfect. 
@shahor02 , do you want to remove the check on the run number?

Reply:
Actually, no: the 521889 is the run from which masking or squashing makes sense at all, before the masking was applied on CTF creation level.